### PR TITLE
feat: make chart.yaml modification optional

### LINKED ISSUE #13: Make patching of Chart.yaml completely optional
Helm charts can be published based only in Git version tags without need to parch Chart.yaml at all.

Examples

* Helm < 3.7, https://github.com/eshepelyuk/apicurio-registry-helm/blob/main/.releaserc.yaml
* Helm >= 3.7, https://github.com/schemahero/schemahero-helm/blob/main/.releaserc.yaml

So, this plugin should allow skip changing Chart.yaml at all, when chart version maintained exclusively via tag tags.

### PR DESCRIPTION
**Pull request by eshepelyuk**
Resolves #13

Signed-off-by: Ievgenii Shepeliuk <eshepelyuk@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,52 +1,32 @@
...
-- major
-- premajor
-- minor
-- preminor
-- patch
-- prepatch
-- prerelease
+| Step               | Description                                                  |
+|--------------------|--------------------------------------------------------------|
+| `verifyConditions` | Verify required configuraiotn and login to Helm registry.    |
```
Comment (m1pl) [inline README.md:10]: typo: configuration

**Review by m1pl — CHANGES_REQUESTED** [review]
Thank you for your contribution. Unfortunately, I can't merge this as it would break existing deployments. You swapped `version` and `appVersion`, which could go wrong in many different ways.

In semantic-release:
- `context.nextRelease.version` is the version of the app as computed by semantic-release

In Helm:
- `appVersion` is the version of the app, which should be equivalent to `context.nextRelease.version`
- `version` is the version of the Helm chart, which can differ from the version of the app

That's why I'm setting `appVersion` to `context.nextRelease.version` and computing the increase for `version`. You are doing it the other way around, which will produce unexpected results, e.g.:

current appVersion: 1.2.3
current version (of the chart): 5.0.0

Let's say the next release is a minor bump, which should result in appVersion 1.3.0 (as computed by semantic release) and chart version 5.1.0. With the code in this PR, the chart version will change from 5.0.0->1.3.0, which will probably cause many tools to not see this as an upgrade and fail.

If you need the flag to not modify the _Chart.yaml_, I'll gladly merge it once you've reverted it to the original version computing logic.

EDIT: There should also be a default value of `true` for the flag, to make this change backward compatible.

File: README.md (hunk 1 above)
Comment (eshepelyuk) [reply README.md:10]: If one keeps both app and chart in the same repo with semantic versioning, it means the semantic release release both as a single artifact, the version of both app and chart must be the same, there is no absolutely good reason to separate them.

If one keeps app outside the chart, than appVersion never must be changed manually.
That is the idea of a change.

Comment (eshepelyuk) [reply README.md:10]: Closing this.

Comment (m1pl) [reply README.md:10]: @eshepelyuk I agree with you. Nevertheless there are many people who have diverging versions for whatever reasons. This change will be breaking things for them, which is why I can't merge it as is.

**Pull request closed without merging**